contracts/contracts/GrantRound.sol

```diff
@@ -0,0 +1,134 @@
+pragma solidity ^0.8.5;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "./GrantRegistry.sol";
+
+contract GrantRound {
+  using SafeERC20 for IERC20;
```

Comment:
nit: Let's put a line of whitespace between each variable declaration, unless anyone is highly opposed to this. I think it's much more readable with some breathing room.